charts/apisix-ingress-controller/templates/configmap.yaml

```diff
@@ -58,7 +58,11 @@ data:
       {{ else }}
       default_cluster_base_url: http://{{ .Values.config.apisix.serviceName }}.{{ .Values.config.apisix.serviceNamespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.config.apisix.servicePort }}/apisix/admin
       {{- end}}
+      {{- if .Values.config.apisix.existingSecret }}
+      default_cluster_admin_key: "{{"{{"}}.DEFAULT_CLUSTER_ADMIN_KEY{{"}}"}}"
```

Comment:
If ENV is used for configuration, this field is not required in the configuration file.

Reply:
done, thanks